cf-check/ts_key_read.c

```diff
@@ -0,0 +1,85 @@
+#include <platform.h>
+#include <ts_key_read.h>
+#include <logging.h>
+#include <definitions.h> // CF_BUFSIZE
+#include <known_dirs.h> // GetStateDir()
+#include <file_lib.h> // FILE_SEPARATOR
+#include <alloc.h> // xstrdup()
+
+void ReadMonitoringSlotsMeta(const char *ts_key_path, Seq *slotsMeta)
```

Comment:
Reference the copied from file: libpromises/monitoring_read.c and vice-versa so that when/if we refactor the hard-coded limit of CF_OBSERVABLES at 100 we might refactor this code to be shared from one common place.